system/Helpers/url_helper.php

```diff
@@ -522,26 +522,31 @@ function auto_link(string $str, string $type = 'both', bool $popup = false): str
 if (! function_exists('prep_url'))
 {
 	/**
-	 * Prep URL - Simply adds the http:// part if no scheme is included.
+	 * Prep URL - Simply adds the http:// or https:// part if no scheme is included.
 	 *
 	 * Formerly used URI, but that does not play nicely with URIs missing
 	 * the scheme.
 	 *
-	 * @param  string $str the URL
+	 * @param  string  $str    the URL
+	 * @param  boolean $secure set true if you want to force https://
 	 * @return string
 	 */
-	function prep_url(string $str = ''): string
+	function prep_url(string $str = '', bool $secure = false): string
 	{
-		if ($str === 'http://' || $str === '')
+		if ($str === 'http://' || $str === 'https://' || $str === '//' || $str === '')
```

Comment:
How about use strict `in_array`?

```suggestion
		if (in_array($str, ['http://', 'https://', '//', '']), true)
```